graphql/src/schema/ast.rs

```diff
@@ -411,6 +412,23 @@ pub fn get_input_object_definitions(schema: &Document) -> Vec<InputObjectType> {
         .collect()
 }
 
+/// Determines if the field is an imported type
+pub fn is_field_imported(
```

Comment:
This would also be really nice to put into the extension trait on `Document` I mentioned above.